scripts/scripts.js

```diff
@@ -21,7 +20,6 @@ import {
 const LCP_BLOCKS = ['hero']; // add your LCP blocks to the list
 const TEMPLATE_LIST = {
   article: 'article',
-  'hub-l2': 'hub',
```

Comment:
Why was this removed?

Reply:
I don't know - I ran unto a merge conflict when I tried to merge main back into this branch before issuing the PR.  Probably was changed in the resolution, though I don't recall that particular line being in conflict at all.  The update I made in this file was to remove all the video handling code.  I suppose I can pull down script.js from main and redo the changes manually instead of letting Git figure out what to do, just to be safe.